src/Form/AuthenticationForm.php

```diff
@@ -106,6 +108,13 @@ class AuthenticationForm extends ConfigFormBase {
    */
   protected $fileSystem;
```

Comment:
$fileSystem became an unused variable unless you change your implementation as I suggested below.

Reply:
removed